packages/verticaltabs/src/react/__specs__/index.spec.tsx

```diff
@@ -16,6 +16,14 @@ describe('VerticalTabs', () => {
     expect(ref).not.toBeNull()
   })
 
+  it('composes className', () => {
+    const { container } = render(<VerticalTabs className="compose-classname" />)
+
+    expect(container.firstChild).toHaveClass(
+      'psds-verticaltabs compose-classname'
+    )
+  })
+
```

Comment:
Please add className compose tests for the subcomponents that are exposed as named exports or are attached to `VerticalTabs` function.